src/qtui/playlist-qt.cc

```diff
@@ -60,6 +61,7 @@ PlaylistWidget::PlaylistWidget (QWidget * parent, Playlist playlist) :
 
     updateSettings ();
     header->updateColumns ();
+    setSortingEnabled (true);
```

Comment:
Sorting needs to be implemented in such a way that the new order makes it down to the underlying playlist.  I don't think that this approach does so?

Reply:
I think that setSortingEnabled() is meant to change only the view and not the underlying model, which is not what we want here.  What we'll probably need to do is intercept the header clicks, map those to the matching Playlist::SortType value, and call into libaudcore (Playlist::sort_by_tuple) to do the actual sorting.